source/includes/fundamentals/code-snippets/mcs.java

```diff
@@ -134,4 +134,20 @@ private static void createSslSettings() {
             System.out.print("---------------------------------------");
         }
     }
+
+    private static void createLoggerSettings() {
+        try {
+            ////begin LoggerSettings
+            MongoClient mongoClient = MongoClients.create(
+                MongoClientSettings.builder().applyConnectionString(new ConnectionString("<your connection string>"))
+                .applyToLoggerSettings(builder ->
+                     builder.maxDocumentLength(5000))
```

Comment:
Comment: in the ticket comments, I noticed that the number parameter was formatted with a _ as 10_000. Should this value be `5_000`?

Reply:
Underscores in numbers is a [Java 7 feature](https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html). 5000 == 5_000, so either is fine. The latter is used for readability.

Reply:
Learn something every day. Adjusted.